src/components/map-container.js

```diff
@@ -468,22 +479,20 @@ export default function MapContainerFactory(MapPopover, MapControl) {
           >
             {this._renderDeckOverlay(layersToRender)}
             {this._renderMapboxOverlays(layersToRender)}
-            {/*
-                By placing the editor in this map we have to perform fewer checks for css zIndex
-                and fewer updates when we switch from edit to read mode
-              */}
             <Draw
               datasets={datasets}
-              editor={uiState.editor}
-              features={editor.features}
+              editor={editor}
+              filters={this.polygonFilters(this.props)}
               isEnabled={isEdit}
               layers={layers}
```

Comment:
Should we show layer that is currently hidden? It's probably better to only show visible layer. You can achieve that by passing `layersToRender`, it is a `{[layer.id]: layer}` Object

Reply:
layersToRender container {[layer.id]: true|false}.
It looks like the current pattern is to pass both __layers__ and __layersToRender__ so I need to pass both layers and layers to render